src/mercury_engine_data_structures/formats/bmsem.py

```diff
@@ -0,0 +1,38 @@
+import construct
+from construct import (
+    Const,
+    Construct,
+    Float32l,
+    Hex,
+    Int32ul,
+    Struct,
+)
+
+from mercury_engine_data_structures.common_types import StrId, make_vector
+from mercury_engine_data_structures.formats import BaseResource
+from mercury_engine_data_structures.game_check import Game
+
+BMSEM = Struct(
+    _magic=Const(b"MSEM"),
+    _version=Const(0x00030001, Hex(Int32ul)),
+    groups=make_vector(Struct(
+            "group_name" / StrId,
+            "layers" / make_vector(Struct(
+                "layer_name" / StrId,
+                "entries" / make_vector(Struct(
+                    "collision_camera" / StrId,
+                    "song" / StrId,             # Is empty if cc is "default".
+                    "unk1" / Float32l,          # Always either 2.0 or 1.5
+                    "unk2" / Float32l,          # Always same number as unk1
```

Comment:
Are unk1 and unk2 maybe again fade_in and fade_out?

Reply:
Maybe. Lemme try to test that later during the day, if I don't manage to I'll just merge since you approved it.

Reply:
I tried setting all values in surface east to `999999999`, `-999999999, `0`, `1` respectively for surface east, and then walked around in a bunch of places. Couldn't hear any difference.
Maybe they're unused, maybe not. But i'll just merge.